build.gradle

```diff
@@ -33,13 +32,15 @@ repositories {
 }
 
 dependencies {
-    compileOnly([group: 'com.mendix', name: 'public-api', version: "$mendixPublicApiVersion"])
-    testCompileOnly("com.mendix:public-api:$mendixPublicApiVersion")
+    compileOnly([group: 'com.mendix', name: 'public-api', version: "(,$mxPublicApiVersion]"])
+    testCompileOnly("com.mendix:public-api:(,$mxPublicApiVersion]")
 
     testImplementation('com.mendix.util:junit-helper:1.0.0') {
         exclude group: 'com.mendix', module: 'public-api'
     }
 
+    implementation(fileTree(mxRuntimeBundles).include('*.jar'))

```

Comment:
Why was this added? Was it needed somewhere?

Reply:
yep, dependencies would not resolve without that when running compose